[Bug] kotlin 枚举字段生成异常

**Describe the bug**

返回对象包含kotlin枚举字段时返回值会生成失败

**Codes**
- AccountController
```
/**
 * 查询当前账户
 * @return 当前账户
 */
@GetMapping
fun findCurrent(): Account {
    return SecurityUtil.currentAccountOrThrow().eraseSecret()
}
```
- Account
```
@Document
class Account : Serializable, BaseEntity() {
    @Id var id: String? = null
    var nickname: String? = null
    var username: String? = null
    /**
     * 性别
     * @see Sex
     */
    var sex = Sex.UNKNOWN
    fun eraseSecret(): Account {
        secret = null
        return this
    }
}
```
- Sex
```
enum class Sex(val description: String) {
    FEMALE("女"), MALE("男"), UNKNOWN("未知");
}
```

如上面代码所示，去掉sex字段则没有问题。

**Logs**
```
run easy-yapi
[INFO]	Start find apis...
[TRACE]	find config file:D:\workspace\github\Server\.easy.api.config
[DEBUG]	use recommend config
[INFO]	search api from: xxx.xxxxx.server.controller.AccountController
[TRACE]	find config file:D:\workspace\github\Server\.easy.api.config
[DEBUG]	use recommend config
[TRACE]	find config file:D:\workspace\github\Server\.easy.api.config
[DEBUG]	use recommend config
[INFO]	search api from: xxx.xxxxx.server.controller.AccountController
[INFO]	[api] before parse class:xxx.xxxxx.server.controller.AccountController
[INFO]	[api] before parse method:xxx.xxxxx.server.controller.AccountController#findCurrent
[INFO]	[json] before parse class:xxx.xxxxx.server.entity.Account
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#id
[INFO]	[json] after parse field:xxx.xxxxx.server.entity.Account#id
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#nickname
[INFO]	[json] after parse field:xxx.xxxxx.server.entity.Account#nickname
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#username
[INFO]	[json] after parse field:xxx.xxxxx.server.entity.Account#username
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#email
[INFO]	[json] after parse field:xxx.xxxxx.server.entity.Account#email
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#bio
[INFO]	[json] after parse field:xxx.xxxxx.server.entity.Account#bio
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#avatar
[INFO]	[json] after parse field:xxx.xxxxx.server.entity.Account#avatar
[INFO]	[json] before parse field:xxx.xxxxx.server.entity.Account#sex
[ERROR]	error to parse body
[TRACE]	java.lang.reflect.UndeclaredThrowableException
	at com.sun.proxy.$Proxy311.resolveEnumFields(Unknown Source)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.parseEnumConstant(AbstractPsiClassHelper.kt:922)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.resolveEnumOrStatic(AbstractPsiClassHelper.kt:767)
	at com.itangcent.intellij.psi.DefaultPsiClassHelper.doGetTypeObject(DefaultPsiClassHelper.kt:205)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetTypeObject(AbstractPsiClassHelper.kt:281)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.parseFieldOrMethod(AbstractPsiClassHelper.kt:1045)
	at com.itangcent.intellij.psi.DefaultPsiClassHelper.parseFieldOrMethod(DefaultPsiClassHelper.kt:316)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper$doGetFields$2.invoke(AbstractPsiClassHelper.kt:582)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper$doGetFields$2.invoke(AbstractPsiClassHelper.kt:29)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.foreachField(AbstractPsiClassHelper.kt:623)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetFields(AbstractPsiClassHelper.kt:564)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetTypeObject(AbstractPsiClassHelper.kt:520)
	at com.itangcent.intellij.psi.DefaultPsiClassHelper.doGetTypeObject(DefaultPsiClassHelper.kt:223)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.doGetTypeObject(AbstractPsiClassHelper.kt:281)
	at com.itangcent.intellij.psi.AbstractPsiClassHelper.getTypeObject(AbstractPsiClassHelper.kt:133)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.parseResponseBody(RequestClassExporter.kt:763)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.processResponse(RequestClassExporter.kt:348)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.exportMethodApi(RequestClassExporter.kt:193)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter.access$exportMethodApi(RequestClassExporter.kt:45)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter$doExport$1$1.invoke(RequestClassExporter.kt:146)
	at com.itangcent.idea.plugin.api.export.core.RequestClassExporter$doExport$1$1.invoke(RequestClassExporter.kt:45)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1$1.invoke(ClassApiExporterHelper.kt:142)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1$1.invoke(ClassApiExporterHelper.kt:33)
	at com.itangcent.intellij.context.ActionContext$callInReadUI$1$1.invoke(ActionContext.kt:461)
	at com.itangcent.common.concurrent.ValueHolder.compute(Holder.kt:100)
	at com.itangcent.intellij.context.ActionContext$callInReadUI$1.run(ActionContext.kt:459)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:59)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:933)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:68)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:58)
	at com.itangcent.intellij.context.ActionContext.callInReadUI(ActionContext.kt:452)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1.invoke(ClassApiExporterHelper.kt:141)
	at com.itangcent.idea.plugin.api.ClassApiExporterHelper$foreachMethod$1$1.invoke(ClassApiExporterHelper.kt:33)
	at com.itangcent.intellij.context.ActionContext$runAsync$2.run(ActionContext.kt:220)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.GeneratedMethodAccessor1501.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.itangcent.common.spi.ProxyBean$invoke$ret$1.invoke(ProxyBean.kt:21)
	at com.itangcent.common.spi.ProxyBean$invoke$ret$1.invoke(ProxyBean.kt:9)
	at com.itangcent.common.utils.ReflectKitKt.privileged(ReflectKit.kt:349)
	at com.itangcent.common.spi.ProxyBean.invoke(ProxyBean.kt:21)
	at com.itangcent.common.spi.ProxyBean.invoke(ProxyBean.kt:12)
	... 39 more
Caused by: java.lang.NullPointerException: constantText must not be null
	at com.itangcent.intellij.jvm.standard.StandardEnumFieldResolverImpl.resolveWithoutConstruct(StandardEnumFieldResolver.kt:159)
	at com.itangcent.intellij.jvm.standard.StandardEnumFieldResolverImpl.resolveEnumFields(StandardEnumFieldResolver.kt:28)
	at com.itangcent.intellij.jvm.standard.StandardPsiResolver.resolveEnumFields(StandardPsiResolver.kt:320)
	... 47 more
[INFO]	[api] after parse method:xxx.xxxxx.server.controller.AccountController#findCurrent
[INFO]	[api] after parse class:xxx.xxxxx.server.controller.AccountController
```

**Environment (please complete the following information):**
 - OS: Windows
 - IDEA version 2022.2.2
 - Plugin version 2.4.6
 - Yapi version 1.10.2

**Config**
```
#Support for javax.validation annotations
field.required=@"javax.validation.constraints.NotBlank
field.required=@javax.validation.constraints.NotNull
field.required=@javax.validation.constraints.NotEmpty
method.return[#pageReturn]=groovy: "xxx.xxxxxx.server.util.ResultUtil<org.springframework.data.domain.PageImpl<" +  helper.resolveLink(it.doc("pageReturn")) +">>"

#Support for Jackson annotations
field.name=@com.fasterxml.jackson.annotation.JsonProperty#value
field.ignore=@com.fasterxml.jackson.annotation.JsonIgnore#value

#yapi status
api.status[#undone]=undone

#deprecated info(java)
field.doc[#deprecated]=groovy:"\n「已废弃」" + it.doc("deprecated")
field.doc[@java.lang.Deprecated]=「已废弃」

#deprecated info(kotlin)
field.doc[@kotlin.Deprecated]=groovy:"\n「已废弃」" + it.ann("kotlin.Deprecated","message")

#Support for Jackson annotations
field.ignore=@com.fasterxml.jackson.annotation.JsonIgnore#value

method.return.main[groovy:it.returnType().isExtend("xxx.xxxxx.server.util.ResultUtil")]=data

json.rule.convert[org.springframework.data.domain.Page]=org.springframework.data.domain.PageImpl
```
```
{
    "methodDocEnable": false,
    "genericEnable": false,
    "feignEnable": false,
    "jaxrsEnable": true,
    "actuatorEnable": false,
    "pullNewestDataBefore": false,
    "postmanToken": "",
    "postmanExportMode": "COPY",
    "postmanCollections": "",
    "postmanBuildExample": true,
    "wrapCollection": false,
    "autoMergeScript": false,
    "postmanJson5FormatType": "EXAMPLE_ONLY",
    "queryExpanded": true,
    "formExpanded": true,
    "readGetter": false,
    "readSetter": false,
    "inferEnable": false,
    "inferMaxDeep": 4,
    "selectedOnly": false,
    "yapiServer": "https://xxx/",
    "yapiTokens": "#\r\n#Fri May 27 19:03:17 CST 2022\r\nserver.main\u003dxxxxxxxxxxxxxxxxxxxxxxxxxx\n",
    "enableUrlTemplating": true,
    "switchNotice": true,
    "loginMode": false,
    "yapiExportMode": "ALWAYS_UPDATE",
    "yapiReqBodyJson5": false,
    "yapiResBodyJson5": false,
    "httpTimeOut": 5,
    "trustHosts": [
        "https://raw.githubusercontent.com/tangcent",
        "https://api.getpostman.com",
        "https://localhost",
        "http://localhost",
        "https://127.0.0.1",
        "http://127.0.0.1"
    ],
    "useRecommendConfig": true,
    "recommendConfigs": "Fastjson,support_mock_for_javax_validation,deprecated_kotlin,yapi_status,spring.validations,Gson,support_mock_for_general,Jackson,spring_Entity,deprecated_java,-enum_auto_select_field_by_type,enum_use_name,ignore,yapi_tag,module,spring.ui,is_file,ignore_transient_field,converts,yapi_tag_kotlin,yapi_mock,ignore_serialVersionUID,properties,javax.validation",
    "logLevel": 50,
    "logCharset": "follow system",
    "outputDemo": true,
    "outputCharset": "follow system",
    "markdownFormatType": "SIMPLE",
    "builtInConfig": ""
}
```

--- a/common-api/build.gradle
+++ b/common-api/build.gradle
@@ -26,7 +26,7 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
 
-    implementation('com.itangcent:commons:1.3.9') {
+    implementation('com.itangcent:commons:1.4.2') {
         exclude group: 'com.google.inject'
         exclude group: 'com.google.code.gson'
     }

--- a/idea-plugin/build.gradle
+++ b/idea-plugin/build.gradle
@@ -78,17 +78,17 @@ dependencies {
 //    implementation fileTree(dir: 'libs', include: ['*.jar'])
 
 
-    implementation('com.itangcent:intellij-idea:1.3.9') {
+    implementation('com.itangcent:intellij-idea:1.4.2') {
         exclude group: 'com.google.inject'
         exclude group: 'com.google.code.gson'
     }
 
-    implementation('com.itangcent:intellij-kotlin-support:1.3.9') {
+    implementation('com.itangcent:intellij-kotlin-support:1.4.2') {
         exclude group: 'com.google.inject'
         exclude group: 'com.google.code.gson'
     }
 
-//    implementation('com.itangcent:intellij-scala-support:1.3.9') {
+//    implementation('com.itangcent:intellij-scala-support:1.4.2') {
 //        exclude group: 'com.google.inject'
 //        exclude group: 'com.google.code.gson'
 //    }
@@ -122,7 +122,7 @@ dependencies {
     // https://mvnrepository.com/artifact/org.mockito/mockito-inline
     testImplementation group: 'org.mockito', name: 'mockito-inline', version: '3.11.0'
 
-    testImplementation('com.itangcent:intellij-idea-test:1.3.9') {
+    testImplementation('com.itangcent:intellij-idea-test:1.4.2') {
         exclude group: 'com.nhaarman.mockitokotlin2', module: 'mockito-kotlin'
     }
 }

--- a/idea-plugin/src/main/resources/.recommend.easy.api.config
+++ b/idea-plugin/src/main/resources/.recommend.easy.api.config
@@ -531,11 +531,14 @@ field.name=@com.alibaba.fastjson.annotation.JSONField#value
 #[enum_auto_select_field_by_type]*
 #Auto map enum to a type matched field in it
 enum.use.by.type=true
+json.rule.enum.convert=~#name()
 
 #[enum_use_name]
 #Map enum to it's name
 enum.use.name=true
+json.rule.enum.convert=~#name()
 
 #[enum_use_ordinal]
 #Map enum to it's ordinal
 enum.use.ordinal=true
+json.rule.enum.convert=~#ordinal()

--- a/idea-plugin/src/test/resources/result/com.itangcent.idea.plugin.config.RecommendConfigReaderTest.SimpleRecommendConfigReaderTest.foreach.txt
+++ b/idea-plugin/src/test/resources/result/com.itangcent.idea.plugin.config.RecommendConfigReaderTest.SimpleRecommendConfigReaderTest.foreach.txt
@@ -21,4 +21,5 @@ constant.field.ignore=groovy:it.name()=="serialVersionUID"
 properties.prefix=@org.springframework.boot.context.properties.ConfigurationProperties#prefix
 field.name=@com.alibaba.fastjson.annotation.JSONField#value
 enum.use.by.type=true
+json.rule.enum.convert=~#name()
 ignore=@Ignore

--- a/idea-plugin/src/test/resources/result/com.itangcent.idea.plugin.settings.helper.RecommendConfigLoaderTest.txt
+++ b/idea-plugin/src/test/resources/result/com.itangcent.idea.plugin.settings.helper.RecommendConfigLoaderTest.txt
@@ -37,3 +37,4 @@ properties.prefix=@org.springframework.boot.context.properties.ConfigurationProp
 field.name=@com.alibaba.fastjson.annotation.JSONField#value
 #Auto map enum to a type matched field in it
 enum.use.by.type=true
+json.rule.enum.convert=~#name()

--- a/idea-plugin/src/test/resources/result/com.itangcent.idea.plugin.settings.helper.RecommendConfigSettingsHelperTest.txt
+++ b/idea-plugin/src/test/resources/result/com.itangcent.idea.plugin.settings.helper.RecommendConfigSettingsHelperTest.txt
@@ -37,3 +37,4 @@ properties.prefix=@org.springframework.boot.context.properties.ConfigurationProp
 field.name=@com.alibaba.fastjson.annotation.JSONField#value
 #Auto map enum to a type matched field in it
 enum.use.by.type=true
+json.rule.enum.convert=~#name()

